.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,8 +1,8 @@
 ---
-name: ⚠️ Bug report
+name: "⚠️ Bug report"
```

Comment:
Why is it necessary to put this in quotes?

Reply:
Honestly, not sure it is necessary. I edited this via the Github UI and it automatically added those in. I can remove if you'd like, but I don't see any issue with including them.

Reply:
just my pet peeve. I don't understand why it defaults to adding quotes around strings that don't need to be quoted. w/e haha